docs/content/stable/admin/yb-admin.md

```diff
@@ -1969,30 +1969,30 @@ To check if any tables are configured for replication, use [list_cdc_streams](#l
 Use the `set_master_addresses` subcommand to replace the source master address list. Use this if the set of masters on the source changes:
 
 ```sh
-yb-admin -master_addresses <target_master_addresses> \
-    alter_universe_replication <source_universe_uuid>_<replication_name> \
-    set_master_addresses <source_master_addresses>
+yb-admin -master_addresses <target-master-addresses> \
+    alter_universe_replication <source-universe-uuid>_<replication-name> \
+    set_master_addresses <source-master-addresses>
```

Comment:
[ignore-errors] is available in all versions. Please add it in all of them

Reply:
For remove_table, right? It's there in all versions.